ucm2/codecs/wcd934x/HeadphoneMicEnableSeq.conf

```diff
@@ -1,3 +1,5 @@
-cset "name='AMIC MUX0' ADC2"
-cset "name='ADC2 Volume' 12"
-cset "name='ADC MUX0' AMIC"
+EnableSequence [
+	cset "name='AMIC MUX0' ADC2"
+	cset "name='ADC2 Volume' 12"
```

Comment:
Boot sequence..

Reply:
thanks, will clean this up for rest of wcd934x codec.